### DISCUSSION TITLE
Unable to get meeting to publish

### DISCUSSION THREAD

**Discussion by jessebrede** [code4recovery/12-step-meeting-list#1593] [General]
Chattanooga AA is having an issue publishing a meeting. Can you tell me why this meeting won't publish? I get no errorrs. 

![Edit-Meeting-“-Word-by-Word-Meeting-–-Online”-‹-Chattanooga-Area-Intergroup-Central-Office-—-WordPress-12-19-2024_02_35_PM](https://github.com/user-attachments/assets/fc9d1555-b0ef-4c51-9831-7b5dae9c3403)
![Screenshot (79)](https://github.com/user-attachments/assets/f4cef8b6-92a7-4305-b59b-d50f86166917)
![Screenshot (80)](https://github.com/user-attachments/assets/35ffa4b8-e475-4ff7-93ac-c09ecd711f9b)
![Screenshot (81)](https://github.com/user-attachments/assets/ababd840-506b-456e-9bcb-fee975279ba3)

**Comment by joshreisner** [comment]
this looks like this should be an online meeting? i'd:

* set `location` to `Chattanooga TN Online Meeting`
* set the address to `Chattanooga, TN`
* set `can i attend i person` to `no`

**Comment by jessebrede** [comment]
That was it. Can we get some error alerts for this case, please? Thank you for all the help and all you do.

**Reply by joshreisner** [reply]
yes, i'll create an issue
